Assets/HotmaxEditorVR/handColliderFollow.cs

```diff
@@ -0,0 +1,66 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class handColliderFollow : MonoBehaviour
+{
+    //--local refs
+    private stateManager _stateManagerMutatorRef;
+    private bool _selectedObjectIsActive = stateManager.selectedObjectIsActive;
+
+    private GameObject collidedWithHand;
+
+    Transform selectorHand;
+
+    private bool isCurrentlyColliding = false;
+
+    private void Awake()
+    {
+        _stateManagerMutatorRef = GameObject.FindObjectOfType(typeof(stateManager)) as stateManager;
+
+        stateManager.selectedObjectIsActiveEvent += updateSelectedObjectIsActive;
+
+        selectorHand = GameObject.Find("Hand2").transform;
```

Comment:
probably should be something less brittle here... are we defining Hand2 in another class we can grab here?

Reply:
got it